echopype/tests/echodata/test_echodata.py

```diff
@@ -466,17 +440,68 @@ def test_nan_range_entries(range_check_files):
     assert xr.Dataset.equals(nan_locs_backscatter_r, nan_locs_Sv_range)
 
 
-def test_update_platform(update_platform_samples):
-    raw_file, extra_platform_data_file = update_platform_samples
-    extra_platform_data_file_name = extra_platform_data_file.name
-
-    ed = echopype.open_raw(raw_file, "EK80")
+@pytest.mark.parametrize(
+    ["ext_type", "sonar_model", "updated", "path_model", "raw_path", "platform_data"],
+    [
+        (
+                "external-trajectory",
+                "EK80",
+                ("pitch", "roll", "longitude", "latitude"),
+                "EK80",
```

Comment:
It seems that the entire block is shifted 1 tab more than needed to the right. Not crucial at all, just the line overflows my screen.

Reply:
Yeah, I saw that. I think PyCharm did it. I figured pre-commit would've fixed it, but it looks like it didn't.